Cargo.toml

```diff
@@ -21,15 +21,18 @@ use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
 secp-endomorphism = ["secp256k1/endomorphism"]
 secp-lowmemory = ["secp256k1/lowmemory"]
 secp-recovery = ["secp256k1/recovery"]
+std = ["secp256k1/std", "bitcoin_hashes/std"]
+use-bare-io = ["bare-io", "bitcoin_hashes/bare-io"]
 
 [dependencies]
 base64-compat = { version = "1.0.0", optional = true }
 bech32 = "0.7.2"
-bitcoin_hashes = "0.9.0"
+bitcoin_hashes = { git = "https://github.com/justinmoon/bitcoin_hashes.git", branch = "more_no_std", features = [], default-features = false }
```

Comment:
[Hacky fork to get things working](https://github.com/rust-bitcoin/bitcoin_hashes/compare/master...justinmoon:more_no_std)